### PR TITLE
Added XE8 features: HTML pagination, view website, and voice commands.

### PR DESCRIPTION
**Pull request by allevato**

### CODE REVIEW

File: main_handler.py [hunk 1]
```diff
@@ -40,11 +40,24 @@
     loader=jinja2.FileSystemLoader(os.path.dirname(__file__)))
 
 
+PAGINATED_HTML = ("<article class='auto-paginate'>"
```
Comment (alainv) [inline main_handler.py]: Use triple quotes instead of parenthesis:

PAGINATED_HTML = """
<article> ....
"""

File: main_handler.py [hunk 2]
```diff
@@ -73,7 +86,7 @@ def _render_template(self, message=None):
     # self.mirror_service is initialized in util.auth_required.
     try:
       template_values['contact'] = self.mirror_service.contacts().get(
-        id='Python Quick Start').execute()
+        id='python-quick-start').execute()
```
Comment (alainv) [inline main_handler.py:91]: This change is backward compatible... How will it work for existing users revisiting the Glassware?

Comment (allevato) [inline main_handler.py:91]: This change was needed to add TAKE_A_NOTE to the sample, due to the way the contact's ID is processed (it can't have spaces). You're right that existing users will get a duplicate contact because of it; it'll look a little janky but one option would be to just have the bootstrapper delete the contact "Python Quick Start" always. Sound good?

Comment (alainv) [inline main_handler.py:91]: SGTM, you can do that in another CL.

File: main_handler.py [hunk 3]
```diff
@@ -40,11 +40,26 @@
...
+    <p>Cats are <em class='yellow'>solar-powered.</em> The time they spend
+    napping in direct sunlight is necessary to regenerate their internal
+    batteries. Cats that do not receive sufficient charge may exhibit the
+    following symptoms: lethargy, irritability, and disdainful glares. Cats
+    will reactivate on their own automatically after a complete charge
+    cycle; it is recommended that they be left undisturbed during this
+    process to maximize your enjoyment of your cat.</p><br/><p>
+    For more cat maintenance tips, tap to view the website!</p>
+    </article>
+    """
```
Comment (alainv) [inline main_handler.py]: No need for the 4-space indentation.

Comment (allevato) [inline main_handler.py]: Fixed.
